### PR TITLE
Docs: Update timeseries axis docs for 10.2

### PR DESCRIPTION
**Pull request by nmarrs**
Update timeseries axis docs for 10.2

Update doc ordering based on current UI

Related changes are:
https://github.com/grafana/grafana/pull/74117

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 1]
```diff
@@ -178,6 +178,10 @@ The following example shows three series: Min, Max, and Value. The Min and Max s
 
 Options under the axis category change how the X and Y axes are rendered. Some options do not take effect until you click outside of the field option box you are editing. You can also or press `Enter`.
 
+### Time zone
+
+Set the desired time zone(s) to display along the X-axis.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:183]:
```suggestion
Set the desired time zone(s) to display along the x-axis.
```

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 2]
```diff
@@ -199,22 +203,46 @@ Set a fixed width of the axis. By default, Grafana dynamically calculates the wi
...
 By setting the width of the axis, data with different axes types can share the same display proportions. This setting makes it easier for you to compare more than one graph’s worth of data because the axes are not shifted or stretched within visual proximity to each other.
 
-### Soft min and soft max
+### Show grid lines
 
-Set a **Soft min** or **soft max** option for better control of Y-axis limits. By default, Grafana sets the range for the Y-axis automatically based on the dataset.
+Set the axis grid line visibility.
 
-**Soft min** and **soft max** settings can prevent blips from turning into mountains when the data is mostly flat, and hard min or max derived from standard min and max field options can prevent intermittent spikes from flattening useful detail by clipping the spikes past a specific point.
+- **Auto:** Automatically shows grid lines based on the density of the data.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:210]:
```suggestion
- **Auto:** Automatically show grid lines based on the density of the data.
```

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 3]
```diff
@@ -199,22 +203,46 @@ Set a fixed width of the axis. By default, Grafana dynamically calculates the wi
...
+- **On:** Always show grid lines.
+- **Off:** Never show grid lines.
 
-To define hard limits of the Y-axis, You can set standard min/max options. For more information, refer to [Configure standard options][].
+### Color
 
-![Label example](/static/img/docs/time-series-panel/axis-soft-min-max-7-4.png)
+Set the color of the axis
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:216]:
```suggestion
Set the color of the axis.
```

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 4]
```diff
@@ -199,22 +203,46 @@ Set a fixed width of the axis. By default, Grafana dynamically calculates the wi
...
 
 Set the Y-axis values scale.
 
 - **Linear:** Divides the scale into equal parts.
 - **Logarithmic:** Use a logarithmic scale. When you select this option, a list appears for you to choose a binary (base 2) or common (base 10) logarithmic scale.
+- **Symlog:** Use a symmetrical logarithmic scale. When you select this option, a list appears for you to choose a binary (base 2) or common (base 10) logarithmic scale. The linear threshold option allows you to set the threshold at which the scale changes from linear to logarithmic.
+
+### Centered zero
+
+Set the Y-axis to be centered on zero.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:235]:
```suggestion
Set the y-axis to be centered on zero.
```

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 5]
```diff
@@ -199,22 +203,46 @@ Set a fixed width of the axis. By default, Grafana dynamically calculates the wi
...
+
+### Soft min and soft max
+
+Set a **Soft min** or **soft max** option for better control of Y-axis limits. By default, Grafana sets the range for the Y-axis automatically based on the dataset.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:239]:
```suggestion
Set a **Soft min** or **soft max** option for better control of y-axis limits. By default, Grafana sets the range for the y-axis automatically based on the dataset.
```

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 6]
```diff
@@ -199,22 +203,46 @@ Set a fixed width of the axis. By default, Grafana dynamically calculates the wi
...
+
+**Soft min** and **soft max** settings can prevent blips from turning into mountains when the data is mostly flat, and hard min or max derived from standard min and max field options can prevent intermittent spikes from flattening useful detail by clipping the spikes past a specific point.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:241]: "Blip" is a bit colloquial. Would this work? 

I'm not entirely sure how the two parts of this sentence go together. Generally this structure is meant to show a contrast. Also this sentence is super long, so I've tried to chop it up and add some contrast wordingI'm not sure if that works though.

```suggestion
**Soft min** and **soft max** settings can prevent small variations from looking very large when the data is mostly flat. In contrast, hard min or max values derived from standard min and max field options help prevent intermittent spikes from flattening useful detail by clipping the spikes past a specific point.
```

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 7]
```diff
@@ -199,22 +203,46 @@ Set a fixed width of the axis. By default, Grafana dynamically calculates the wi
...
+
+To define hard limits of the Y-axis, You can set standard min/max options. For more information, refer to [Configure standard options][].
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:243]:
```suggestion
To define hard limits of the y-axis, set standard min/max options. For more information, refer to [Configure standard options][].
```

**Review by imatwawana — COMMENTED** [review]
Mostly wording/style edits but I made changes to one bit that you'll have to take a look at.

File: docs/sources/panels-visualizations/visualizations/time-series/index.md (hunk 1 above)
Comment (nmarrs) [reply docs/sources/panels-visualizations/visualizations/time-series/index.md:183]: re: x / y axis capitalization, would you like me to go through the rest of the time series doc file to lower-case the other instances of `X-axis` + `Y-axis`? I think default in the file before was to capitalize them (which is convention I ended up following for added sections)

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 8]
```diff
@@ -178,11 +178,15 @@ The following example shows three series: Min, Max, and Value. The Min and Max s
 
 Options under the axis category change how the X and Y axes are rendered. Some options do not take effect until you click outside of the field option box you are editing. You can also or press `Enter`.
```
Comment (nmarrs) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:179]: @imatwawana - not sure how we want to handle capitalization of x / y axis in the this case 🤔 maybe still just uncapitalize them always? i.e. `x and y axes` vs `X and Y axes`

Comment (imatwawana) [reply docs/sources/panels-visualizations/visualizations/time-series/index.md:179]: yes 100% please take those capitals away. and it should really be `x- and y-axes`.

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 9]
```diff
@@ -191,30 +195,54 @@ To selectively hide axes, [Add a field override][] that targets specific fields.
...
+
+### Centered zero
+
+Set the y-axis to be centered on zero.
+
+### Soft min and soft max
+
+Set a **Soft min** or **soft max** option for better control of y-axis limits. By default, Grafana sets the range for the y-axis automatically based on the dataset.
+
+**Soft min** and **soft max** settings can prevent magnifying small variations in the data when it is mostly flat. In contrast, hard min and max values help prevent obscuring useful detail in the data by clipping intermittent spikes past a specific point.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:241]: This looks great!

Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:241]:
```suggestion
Set a **Soft min** or **soft max** option for better control of y-axis limits. By default, Grafana sets the range for the y-axis automatically based on the dataset.

**Soft min** and **soft max** settings can prevent small variations in the data from being magnified when it's mostly flat. In contrast, hard min and max values help prevent obscuring useful detail in the data by clipping intermittent spikes past a specific point.
```

**Review by imatwawana — APPROVED** [review]
One teeny nit. Otherwise LGTM!

File: docs/sources/panels-visualizations/visualizations/time-series/index.md (hunk 8 above)
Comment (nmarrs) [reply docs/sources/panels-visualizations/visualizations/time-series/index.md]: got changed it and fixed last remaining `Y-axes` that I missed from before (just searching `y-axis` 😬

Comment (imatwawana) [reply docs/sources/panels-visualizations/visualizations/time-series/index.md]: thank you! much appreciated!

**Pull request merged**